src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/updateSubmission/UpdateSubmission.java

```diff
@@ -0,0 +1,130 @@
+package uk.gov.companieshouse.overseasentitiesapi.model.updateSubmission;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.OverseasEntitySubmissionDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.updateSubmission.changes.Change;
+
+import java.util.List;
+
+public class UpdateSubmission {
+    @JsonProperty("type")
+    private String type = "OE02";
+
+    @JsonProperty("userSubmission")
+    private OverseasEntitySubmissionDto userSubmission;
+
+    @JsonProperty("dueDiligence")
+    private DueDiligence dueDiligence;
+
+    @JsonProperty("presenter")
+    private Presenter presenter;
+
+    @JsonProperty("filingForDate")
+    private FilingForDate filingForDate;
+
+    @JsonProperty("noChangesInFilingPeriodStatement")
+    private String noChangesInFilingPeriodStatement;
+
+    @JsonProperty("anyBOsOrMOsAddedOrCeased")
+    private String anyBOsOrMOsAddedOrCeased;
+
+    @JsonProperty("beneficialOwnerStatement")
+    private String beneficialOwnerStatement;
+
+    @JsonProperty("changes")
+    private List<Change> changes;
+
+    @JsonProperty("additions")
+    private List<Change> additions;
+
+    @JsonProperty("cessations")
+    private List<Change> cessations;
+
```

Comment:
Same here this might need to be a List of Type Cessation

Reply:
Here's an example of how the target xml needs to be -- for PSCs it would need to include --
<pscCessations>
      <pscs>
        <psc>
          <actionDate>2017-06-14</actionDate>
          <appointmentId>1</appointmentId>
          <appointmentType>1</appointmentType>
          <personName>
            <title>-1</title>
            <titleOther>MEISTER</titleOther>
                        <forename>MISTER</forename>
                        <surname>MISTER</surname>
              </personName>
        </psc>
      </pscs>
    </pscCessations>

And For Officers this would be it --
<resignations>
      <officers>
        <officer>
          <actionDate>2009-03-03</actionDate>
          <corporateOfficer>false</corporateOfficer>
          <personName>
            <surname>WHEARE</surname>
            <forename>THOMAS</forename>
          </personName>
          <birthDate>1944-10-11</birthDate>
        </officer>
      </officers>
</resignations>

Reply:
Please take this out of this PR.